### DISCUSSION TITLE
pay for developing！

### DISCUSSION THREAD

**Discussion by yg2342423523** [Purfview/whisper-standalone-win#278] [Q&A] [answered]
Hello sir, I hope I can pay to develop something related to this, how can I contact you? I wonder if you are willing to do this?

**Comment by Purfview** [comment, marked as answer]
> I wonder if you are willing to do this?

Hi, you can add a message for a small feature you want when you make a donation. [The donation buttons are at the main page]
Or you can send an email for something bigger than a small feature.

> how can I contact you?

purfview [at] protonmail.com
